hypermap/aggregator/tests/mocks/worldmap.harvard.edu/data/search/api

```diff
@@ -907,7 +907,7 @@
          "bbox":{
             "minx":"71.13516205105506",
             "miny":"36.45478671723314",
-            "maxx":"",
+            "maxx":"73.959356679",
```

Comment:
@capooti @ingenieroariel FYI I'm not sure whether this value was blank by design or not, but I filled it in based on its value in http://worldmap.harvard.edu/geonetwork/srv/en/csw?outputschema=http%3A%2F%2Fwww.isotc211.org%2F2005%2Fgmd&service=CSW&request=GetRecordById&version=2.0.2&elementsetname=full&id=1eb32bfd-4b39-457a-b60b-e307d24c6b8f